NOTICE

```diff
@@ -383,3 +416,301 @@ third-party archives.
 
 ---
 
+Eclipse Public License 2.0
+(JUnit 5 (Bill of Materials) 5.9.2)
```

Comment:
Add link to www.junit.org, and I also like the syntax used here that indicates link to maven, so something like `maven:junit:junit:5.9.2` would be cool to add.

Reply:
link added 👍🏻

Reply:
I see the usable-ness of using the gradle coordinates rather than the "$name $version" format. However, isn't this a feature request to the tool we use to generate the file? I don't want to meddle too might with the format, since that would be difficult to maintain.

Reply:
I agree (until we have the fix though, I tend to update the files manually.)